great_expectations/core/expectation_diagnostics/expectation_diagnostics.py

```diff
@@ -215,3 +259,82 @@ def _convert_checks_into_output_message(
         output_message += "\n"
 
         return output_message
+
+    @staticmethod
+    def _check_input_validation(
+        expectation_instance,
+        examples: List[ExpectationTestDataCases],
+    ) -> ExpectationDiagnosticCheckMessage:
+        """Check that the validate_configuration method returns True"""
+        passed = False
+        try:
+            first_test = examples[0]["tests"][0]
+        except IndexError:
+            # No examples, so can't get kwargs for ExpectationConfiguration
+            pass
```

Comment:
Same note as above - let's wrap this comment in a logger statement to help future devs?